### PR TITLE
Added test file for armada/cmd/armadactl/cmd/watch.go

### PR DESCRIPTION
**Pull request by biswajit-9776**
Fixes #
Issue #2167 
#### Special notes for your reviewer:

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-626) by [Unito](https://www.unito.io)

### CODE REVIEW

File: cmd/armadactl/cmd/watch_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+
+	"github.com/armadaproject/armada/internal/armadactl"
+)
+
+func TestWatch(t *testing.T) {
+	// TODO there are no tests for invalid input because cobra silently discards those inputs without raising errors
+	tests := map[string]struct {
+		Flags               []flag
```
Comment (kannon92) [inline cmd/armadactl/cmd/watch_test.go:16]:
```suggestion
		flags               []flag
```

File: cmd/armadactl/cmd/watch_test.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+
+	"github.com/armadaproject/armada/internal/armadactl"
+)
+
+func TestWatch(t *testing.T) {
+	// TODO there are no tests for invalid input because cobra silently discards those inputs without raising errors
```
Comment (kannon92) [inline cmd/armadactl/cmd/watch_test.go:14]: Is this a TODO for this PR?

Comment (biswajit-9776) [reply cmd/armadactl/cmd/watch_test.go]: Sorry for the comment. This was a mistake from my side. The TODO comment was for another file. This PR doesn't include any TODO comments.

File: cmd/armadactl/cmd/watch_test.go (hunk 1 above)
Comment (biswajit-9776) [reply cmd/armadactl/cmd/watch_test.go]: Changes have been applied.

**Review by kannon92 — APPROVED** [review]

**Review by Sharpz7 — APPROVED** [review]

**Pull request merged**